### DISCUSSION TITLE
How to determine the transit rate for each intestinal segment from intestinal transit time?

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#753] [Questions & Problems]
Hi, all

In recent days, for some reasons I am trying to modify an oral PBPK model in another software. In this software, the intestinal segment transit rate is defined as, taking duodenum as example, **(kTransportSmallIntestine*organismLengthDuodenum/organismLengthSmallInstestine)*Duodenum.DrugSolid** and **kTransportSmallIntestine** is equal to **0.693/organismMeanResidenceSmallIntestine**. 

I think assuming duodenum transit time is equal to **SmallIntestineTransittime*organismLengthDuodenum/organismLengthSmallInstestine**, the transit rate constant should be the inverse of that, which result in **kTransportSmallIntestine*organismLengthSmallInstestine/organismLengthDuodenum**, contradicting with the equation in software. 

And I think the mean residence time determins the time at which about 63.2% of initial amount having passed through the compartment and inverse of mean residence time determining the transit rate. Why does organismMeanResidenceSmallIntestine correlate with 0.693 which is often seen in half-life associated expression?

Am I wrong? Which one is reasonable? 

Then, I attempt to refer to the transit rate set in PK-Sim and MoBi, but what I get is:
![image](https://user-images.githubusercontent.com/54141158/77666772-53483f80-6fbc-11ea-958c-09126ebeffeb.png)
![image](https://user-images.githubusercontent.com/54141158/77667074-b6d26d00-6fbc-11ea-997a-3a32708d5dcb.png)
![image](https://user-images.githubusercontent.com/54141158/77667212-dd90a380-6fbc-11ea-8c5d-c72f4377ec90.png)
The **small intestinal transit time** is only used to calculate a factor, not used in duodenum **intestinal transit rate**. However, the **intestinal transit rate** for each GI segment is distinct. How to determine this value? Is it independent with intestine **small intestinal transit time**? 

Thanks for any comment.

**Comment by jzq90** [comment]
The **intestinal transit rate (absolute)** is calculated by **intestinal transit time/ SITT factor**.
![图片](https://user-images.githubusercontent.com/58142266/77735497-9ef2fb80-7045-11ea-80c7-6cab2b3613d6.png)
The **intestinal transit rate (absolute)**  is used in the passive transport building block to simulate the transit in the small intestine. So eventually the real intestinal transit rate is dependent on **intestinal transit rate time**.
![图片](https://user-images.githubusercontent.com/58142266/77735620-dcf01f80-7045-11ea-8d85-e7dd58ecfcb7.png)

**Comment by jzq90** [comment]
However, I also don't know what's the meaning of SITT factor and why divide the intestinal transit rate with this factor

**Comment by StephanSchaller** [comment]
SITT = Small intestine transit time factor

It is a divisor to the rate such that if you increase it, the transit time will be increased (and the rate reduced). It is only a scaling factor for convenience.

**Comment by wangwei1619** [comment]
I am not sure what “intestinal transit rate” actually means. This parameter seems to a large degree different from “intestinal transit rate (absolute)” in value, which is eventually used in simulation, but two parameters are using similar names. I think “intestinal transit rate” is used to make transit rate of each segment distinct. However, to achieve this goal, is it possible to just use anatomic information like length and radius plus intestinal transit time to define transit rate for GI segment？

**Comment by jzq90** [comment]
> It is a divisor to the rate such that if you increase it, the transit time will be increased (and the rate reduced). It is only a scaling factor for convenience.

What makes me confused is that the SITT factor isn't 1 by default, instead, it is calculated by a linear fit with slope and intercept. The slope and intercept are both constants which are dependents on the species but not on demographic data. So the SITT factor keeps the same for all human individuals. Based on this setting, it is very hard to adjust the SITT factor to change the SITT rate or time.

**Comment by Yuri05** [comment]
>So the SITT factor keeps the same for all human individuals. 

This is not correct. The factor is a linear function of the small intestinal transit time. Different individuals have in general different transit time and thus different scaling factors. 
Thus if you change the transit time manually, or create a population where transit time vary between individuals - individuals will have different scaling factors.
S. also #484 and #580
<img width="500" src="https://user-images.githubusercontent.com/25061876/89451051-e27c1580-d75b-11ea-9680-0f11e010ef51.png">
